{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:19: Exception"}, "start": 1594110575752, "stop": 1594110575752, "uuid": "18102a96-296a-4de3-b8a3-c6c613ca4b6b", "historyId": "985c3d972d231f87d0914b89f78754fd", "testCaseId": "cb41bd139e929d9eef92825c451cc27a", "fullName": "test_allure#test_broken", "labels": [{"name": "suite", "value": "test_allure"}, {"name": "host", "value": "heweimaodeMacBook-Pro.local"}, {"name": "thread", "value": "58180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure"}]}